build.ps1

```diff
@@ -0,0 +1,47 @@
+$ClientImportPath="github.com/hazelcast/hazelcast-go-client"
+$ListOfPackages = New-Object System.Collections.ArrayList
+go list $ClientImportPath/... | Select-String -pattern ".*/test|.*/compatibility|.*/rc|.*/sample" -notmatch | %{$ListOfPackages.add($_)}
+$PackageList=$ListOfPackages -join ","
+
+#run linter
+cd $env:gopath\src\$ClientImportPath
+.\linter.ps1
+
+# currently comment out due to a lot of found issues
```

Comment:
Why does it find a lot of issues? Can you show a few of the found issues?

Reply:
I have already written about them to Furkan. You can see those issues here - https://hazelcast-l337.ci.cloudbees.com/job/golang-client-windows-temp/1/consoleFull in time 19:16:15.

Reply:
I think I found why it is finding issues. I will `gometalinter.json` for windows.

Reply:
Great, thanks!